wandering-hour-clock.ino

```diff
@@ -17,14 +17,25 @@
 #include <ArduinoOTA.h>
 #include <TimeLib.h>  // https://github.com/PaulStoffregen/Time
 #include <WebServer.h>
+#include <Preferences.h>
 
 #include <Stepper.h>
+
 #include "secrets.h"
 
 // Replace the ssid and password in secrets.h
 const char* ssid = SECRET_SSID;
 const char* password = SECRET_PASSWORD;
 
+// Hostname
+const char* hostname = "wander";
+
+// Preferences library namespace and keys
+const char* pref_namespace = "whc"; // "Wandering Hour Clock"
```

Comment:
can we change this to `wandering-hour-clock` to keep it backward compatible with existing installs?

Reply:
Thanks for reviewing everything!  I'll make that change now!